### ISSUE TITLE
Vertical USB?

### ISSUE THREAD

**Issue by R-OLE** [sparkfun/LilyPad_LilyMini_ProtoSnap#1]
If there is the intent to sew this board to cloth or other materials, why would you put the USB connector sticking out of the PCB plane?
Seems that would be:
- uncomfortable if pressed against skin
- greatly increase the chances of snagging on supporting material or other "things"
- greatly increase the chances of damaging the PCB or even snapping-off the USB connector

**Comment by R-OLE** [comment]
It would also make it more difficult to provide power via USB cable in an application, as the cable would not be laying down.

**Comment by R-OLE** [comment]
Ooops. Wrong button.

**Comment by mgrusin** [comment]
Thanks for the feedback! Any design is a series of sometimes difficult tradeoffs, and what works for one person may not work for another. Some of the reasons we chose the vertical connector are:

Price was one of the main drivers of this board. The vertical connector uses less real-estate and doesn't interfere with adjacent sewtabs, allowing a smaller and less-expensive board.

We've had considerable trouble in the past with "flat" USB connectors ripping off boards, and wanted to come up with a very strong solution for student use. The vertical connector is half surface-mount and half through-hole, and is extremely strong.

In testing we found that most users will temporarily attach a USB cable to program the board, then use the coin cell to power it. We've also found that the vertical connector is easier to attach a USB cable to in most situations (where the board is attached to the outside of a project). If a straight-out USB cable doesn't work well there are right-angle USB cables available that may solve these problems.

Sincere thanks for the feedback. We're always reevaluating our design choices and doing our best to incorporate what works best for our customers.
